### PR TITLE
eslint fixes - no-unsafe-optional-chaining

### PR DESCRIPTION
**Pull request by Emily-Ke**
Pull request checklist:
 - [x] Features and bugfixes should be PRed into the `develop` branch, **not** `main`
 - [ ] All text is internationalized
 - [ ] There are no linter errors
 - [x] New features support all states (loading, error, etc) 

-----

- Internationalization: I think I internationalized everything in `Paginator.jsx` because it is a small module. I only internationalized the text in `Individuals.jsx` that was closely related to the `no-unsafe-optional-chaining` errors.
- Linter errors: I left some linter errors in `Individuals.jsx` because they are being addressed in #407.

### CODE REVIEW

File: src/pages/individual/Individual.jsx [hunk 1]
```diff
@@ -72,9 +72,10 @@ export default function Individual() {
             src: annotation?.asset_src,
             guid: annotation?.asset_guid,
             altText: annotation?.created
-              ? intl.formatMessage({
-                  id: 'ANNOTATION_CREATED',
-                }) + annotation?.created
+              ? intl.formatMessage(
+                  { id: 'ANNOTATION_CREATED_ON_DATE' },
+                  { date: annotation.created },
+                )
```
Comment (Atticus29) [inline src/pages/individual/Individual.jsx:78]: Thank you! Much better idea.

File: src/pages/individual/Individual.jsx [hunk 2]
```diff
@@ -143,7 +149,7 @@ export default function Individual() {
     );
     const firstNineAssets = slice(uniqueModifiedAssets, 0, 9);
     return firstNineAssets;
-  }, [individualData]);
+  }, [individualData, intl]);
```
Comment (Atticus29) [inline src/pages/individual/Individual.jsx:152]: I hadn't thought that intl would ever change?

**Review by Atticus29 — APPROVED** [review]
LGTM. A question, but otherwise makes sense!

File: src/pages/individual/Individual.jsx (hunk 2 above)
Comment (Atticus29) [reply src/pages/individual/Individual.jsx:152]: (maybe this is a lifecycle question ultimately).

Comment (Emily-Ke) [reply src/pages/individual/Individual.jsx:152]: If we ever allow language switching in the application, it will.

**Pull request merged**